app/routes/labs._index.mdx

```diff
@@ -0,0 +1,293 @@
+{/\* \* Copyright © 2023 United States Government as represented by the
+
+- Administrator of the National Aeronautics and Space Administration.
+- All Rights Reserved.
+-
+- SPDX-License-Identifier: Apache-2.0 \*/}
+
+import {
+  Button,
+  Card,
+  CardBody,
+  CardFooter,
+  CardGroup,
+  CardHeader,
+  CardMedia,
+  GridContainer,
+  Icon,
+  Link,
+  Tag,
+} from '@trussworks/react-uswds'
+
+import ParallaxImage from './labs/parallax/ParallaxImage'
+
+import React from 'react'
+
+import styles from './labs/across.css'
+import mma from './labs/mma.png'
+
+  <div style={{ backgroundColor: '#0c0f18', color: 'white', minHeight: '2000px' }}>
+
+    <ParallaxImage
+      src="https://svs.gsfc.nasa.gov/vis/a010000/a012700/a012740/NeutronStarMerger_Shot1_4KStill4_v01_print.jpg"
+      alt="BNS"
+    />
+
+    <GridContainer className="usa-section" style={{minWidth:'75%'}}>
+
+      <div className="section-head"><h1>Overview</h1></div>
+
+      <div className="container" style={{ marginTop: '0px', display: 'flex', justifyContent: 'space-between' }}>
+        <div className="usa-paragraph text-base-lightest magin-top-0" style={{
```

Comment:
You can remove the `use-paragraph` style because we apply it automatically to all paragraphs in Markdown and MDX files: https://github.com/nasa-gcn/gcn.nasa.gov/blob/main/remix.config.js#L53

```suggestion
        <div className="text-base-lightest magin-top-0" style={{
```

Reply:
Done.